package.json

```diff
@@ -52,7 +52,7 @@
     "superagent": "^1.7.2",
     "through2": "^2.0.1",
     "update-notifier": "^0.6.1",
-    "webtask-bundle": "^2.3.0",
+    "webtask-bundle": "^3.0.0",
```

Comment:
Is it possible that latest version of `webtask-bundle` at npm is `v2.3.0` and not `v3.0.0`?

https://www.npmjs.com/package/webtask-bundle